entity/ProductStoreEntities.xml

```diff
@@ -520,6 +523,32 @@ along with this software (see the LICENSE.md file). If not, see
         </seed-data>
     </entity>
 
+    <extend-entity entity-name="DataDocument" package="moqui.entity.document">
+        <relationship type="many" related="mantle.product.store.ProductStoreDataDocument" short-alias="storeDataDocuments">
+            <key-map field-name="dataDocumentId"/></relationship>
+    </extend-entity>
+    <entity entity-name="ProductStoreDataDocument" package="mantle.product.store">
+        <description>Relationship between the ProductStore and DataDocument Entities meant to replace ProductStore.productDataDocumentId fields and the like. See: https://forum.moqui.org/t/productstore-datadocument-options/303.</description>
+        <field name="productStoreId" type="id" is-pk="true"/>
+        <field name="storeDataDocumentTypeEnumId" type="id" is-pk="true"/>
+        <field name="dataDocumentId" type="id"/>
+
+        <!-- TODO: Are these many relationships? -->
+        <relationship type="one" related="mantle.product.store.ProductStore" short-alias="store"/>
+        <relationship type="one" related="moqui.entity.document.DataDocument" short-alias="dataDocument"/>
+        <relationship type="one" title="StoreDataDocumentType" related="moqui.basic.Enumeration" short-alias="type">
+            <key-map field-name="storeDataDocumentTypeEnumId"/></relationship>
+
+        <seed-data>
+            <moqui.basic.EnumerationType description="Product Store Data Document Type" enumTypeId="ProductStoreDataDocumentType"/>
+            <moqui.basic.Enumeration description="Store - Product Data Document" enumId="PsdtProduct" enumTypeId="ProductStoreDataDocumentType"/>
```

Comment:
This and the ones below it are ProductStore DataDocument types so can remove "Store" and "Data Document" from the descriptions, and because these are used primarily for searching we might as well make that explicit with something like "Product Search"

Reply:
Should be done. Check update.